packages/govuk-frontend/src/govuk/init.jsdom.test.mjs

```diff
@@ -143,6 +175,42 @@ describe('initAll', () => {
       })
     )
   })
+
+  it('executes onError callback on component create if specified', () => {
+    document.body.classList.add('govuk-frontend-supported')
+    document.body.innerHTML = '<div data-module="govuk-accordion"></div>'
+
+    jest.mocked(GOVUKFrontend.Accordion).mockImplementation(() => {
+      throw new Error('Error thrown from accordion')
+    })
```

Comment:
**issue** Could we use the same `MockComponentThatErrors` as we're using in other tests for consitency rather than mocking an actual component? If necessary, the class could happily be put in the test file rather than within an initial `describe`.

Reply:
in initAll though, there's no way of overloading/adding to the `components` variable that is looped through right? I think that's why this approach was done.

Reply:
Oh true, we can't change which components are used in `initAll` indeed 🤦🏻 Let's keep mocking the accordion 😊